### PR TITLE
Update pokemon_transfer_worker.py

### PR DESCRIPTION
**Pull request by HannibalChew**
Grammar correction.

### CODE REVIEW

**Comment by Zgrkaralar** [comment]
👍

File: pokemongo_bot/cell_workers/pokemon_transfer_worker.py [hunk 1]
```diff
@@ -60,7 +60,7 @@ def work(self):
                                                                           best_pokemon['cp'],
                                                                           best_pokemon['iv']), 'green')
 
-                        logger.log("Exchange {} pokemon(s)".format(len(all_pokemons)), "green")
+                        logger.log("Exchanging {} pokemon".format(len(all_pokemons)), "green")
```
Comment (TheSavior) [inline pokemongo_bot/cell_workers/pokemon_transfer_worker.py]: It should really say `Transferring`, right?

Comment (Zgrkaralar) [inline pokemongo_bot/cell_workers/pokemon_transfer_worker.py]: Maybe **Transferring** is better cuz on game always saying transfer transfer 💃

**Comment by maestroi** [comment]
seems good, i think there is no clarification is needed because in the game it is called transfer

**Pull request merged**